phpstan.neon

```diff
@@ -2,6 +2,9 @@ includes:
 	- 'phpstan-baseline.neon'
 
 parameters:
+	level: 'max'
```

Comment:
You are missing to remove the info from the composer.json file

Reply:
Are you sure?

![kép](https://user-images.githubusercontent.com/952007/99266619-3e5a3e00-2823-11eb-8410-a7f3228e95f3.png)